index.js

```diff
@@ -44,7 +44,7 @@ function init(
         port: configWithDefaults.websocketPort,
     });
 
-    setupKeyInput(screen.driver);
+    if(process.stdin.setRawMode) setupKeyInput(screen.driver);
```

Comment:
According to [waveshare FAQ](https://www.waveshare.com/wiki/4.2inch_e-Paper_Module): 'Please set the e-Paper to sleep mode or power off it directly, otherwise, the e-Paper is damaged because of working in high voltage for long time.'

`setupKeyInput:29` ensures that the display is properly put to sleep when the process is exited with control-c. I think we need to make sure we take the same precaution when using process managers.

Reply:
Good point. I've added it with commit c6734e8. If possible I would like to stick with pm2 as a start. I'm currently running my application with pm2 on a Raspberry Pi Zero and can confirm that it takes some additional ressources.

About the implementation I'm not quite sure if it's still needed to check explicitly for the key input within the 'setupKeyInput' function. Otherwise it could be enough to intercept the SIGINT signal (from whatever source).